src/gt4py/backend/dace_backend.py

```diff
@@ -280,7 +282,23 @@ def generate_dace_args(self, ir, sdfg):
 
         symbols = {f"__{var}": f"__{var}" for var in "IJK"}
         for name, array in sdfg.arrays.items():
-            if not array.transient:
+            if array.transient:
```

Comment:
Instead of adding stride symbols for transients here to the interface, they should be set to expressions based on their shape in the above `_specialize_transient_strides` utility.

Reply:
It seems like that method might already be doing that, because I deleted this block and the test is still working. Does that sound right to you?

Reply:
Basically, `symbols` will at the end be a dictionary where for each of the symbols that need to be passed into the cpp dace call (dict key) we provide a string that we can use to get the symbol's value from the SID (dict value). For transient's, that key will never be looked up (since it's not needed to be passed in), so it's just extra entries in the `symbols` that aren't needed. Since the value can however not be meaningful even if they weren't specified (they are not passed in in the bindings, but rather computed based on the domain size), I would still vote for skipping transients here. Maybe an 

```python
if array.transient:
    continue
```
    
Is nicer visually though.